scalajslib/worker/0.6/src/ScalaJSWorkerImpl.scala

```diff
@@ -83,7 +83,8 @@ class ScalaJSWorkerImpl extends ScalaJSWorkerApi {
       sourceMap: Boolean, // ignored in 0.6
       moduleKind: ModuleKind,
       esFeatures: ESFeatures,
-      moduleSplitStyle: ModuleSplitStyle // ignored in 0.6
+      moduleSplitStyle: ModuleSplitStyle, // ignored in 0.6
+      outputPatterns: OutputPatterns // ignored in 0.6
```

Comment:
It makes sense to focus on dropping 0.6 support at this point. See #2218.

Reply:
Yes, this is meant to go very soon, so we don't want to invest time on it :)